src/main/java/Duke.java

```diff
@@ -94,22 +94,22 @@ public static void addToDo(String line){
      *
      * @param line User input
      */
-    public static void addDeadline(String line){
+    public static void createDeadline(String line){
         int startOfDateTime = line.indexOf("by");
         String dateTime = line.substring(startOfDateTime+3);
         String description = line.substring(9,startOfDateTime-1);
 
         Deadline d = new Deadline(description,dateTime);
         addToList(d);
-        System.out.println("(at: "+ d.getDateTime() + ")");
+        System.out.println("(by: "+ d.getDateTime() + ")");
     }
 
     /**
      * Creates a new Event
      *
      * @param line User input
      */
-    public static void addEvent(String line){
+    public static void createEvent(String line){
         int startOfDateTime = line.indexOf("at");
         String dateTime = line.substring(startOfDateTime+3);
```

Comment:
I noticed that when you use "+", sometimes you have spacing before or after it, but sometimes don't. Maybe you can adjust the spacing to make it neat.